components/elm/cime_config/testdefs/testmods_dirs/elm/fates_harvest_carbon/user_nl_elm

```diff
@@ -0,0 +1,72 @@
+!----------------------------------------------------------------------------------
+! Users should add all user specific namelist changes below in the form of 
+! namelist_var = new_namelist_value 
+!
+! Include namelist variables for drv_flds_in ONLY if -megan and/or -drydep options
+! are set in the CLM_NAMELIST_OPTS env variable.
+!
+! EXCEPTIONS: 
+! Set use_cndv           by the compset you use and the CLM_BLDNML_OPTS -dynamic_vegetation setting
+! Set use_vichydro       by the compset you use and the CLM_BLDNML_OPTS -vichydro           setting
+! Set use_cn             by the compset you use and CLM_BLDNML_OPTS -bgc  setting
+! Set use_crop           by the compset you use and CLM_BLDNML_OPTS -crop setting
+! Set spinup_state       by the CLM_BLDNML_OPTS -bgc_spinup      setting
+! Set irrigate           by the CLM_BLDNML_OPTS -irrig           setting
+! Set co2_ppmv           with CCSM_CO2_PPMV                      option
+! Set dtime              with L_NCPL                             option
+! Set fatmlndfrc         with LND_DOMAIN_PATH/LND_DOMAIN_FILE    options
+! Set finidat            with RUN_REFCASE/RUN_REFDATE/RUN_REFTOD options for hybrid or branch cases
+!                        (includes $inst_string for multi-ensemble cases)
+! Set glc_grid           with CISM_GRID                          option
+! Set glc_smb            with GLC_SMB                            option
+! Set maxpatch_glcmec    with GLC_NEC                            option
+! Set glc_do_dynglacier  with GLC_TWO_WAY_COUPLING               env variable
+!----------------------------------------------------------------------------------
+
+ use_cn = .false.
+ do_harvest = .true.
+ wood_harvest_units = 2
+ fates_spitfire_mode = 0
```

Comment:
@sshu88 I started up this elm testmod for carbon harvest to move the fates harvest variables here for future testing.  I have a few questions:
- Should the fire be off in this test?
- No external data is necessary for the carbon harvest correct?
- Can the `ndepdata` be removed?
- Is the full list of variables below necessary, or can it be truncated?

Reply:
@glemieux I currently have not submitted the pull request of carbon based harvest yet. I believe the current work completed the interface to pass the harvested carbon back to ELM/CLM, but not C based harvest. Let me discuss with @jenniferholm if I shall create the pull request?
For answering your questions:
1. Currently I performed all my tests with fire module off. So I agree to turn off the fire module.
2. We will need a land use timeseries with C based harvest rate, instead of area fraction based harvest rate.
3. My current works are based on C-only FATES version, `ndepdata` shall not be a problem if you want to remove it.
4. Yes you can truncate most of the variables since they're not directly relevant to LUC. Let me paste the necessary variables here for your reference:
hist_fincl1 = 'TOTPRODC','PRODUCT_CLOSS','DWT_PROD100C_GAIN','DWT_PROD10C_GAIN','PROD10C','PROD100C','HRV_DEADSTEMC_TO_PROD10C', 'HRV_DEADSTEMC_TO_PROD100C'
hist_fincl2 = 'GPP','NEE','NEP','NPP','TLAI','TOTVEGC','FROOTC','LIVECROOTC','DEADCROOTC','LIVESTEMC','DEADSTEMC','TOTPFTC','AR','HR','PCT_LANDUNIT','PCT_NAT_PFT','TOTLITC','CWDC','DWT_CLOSS','TOTECOSYSC','TOTSOMC'

Reply:
Thanks for the quick feedback.  I think having a quick check in with you and @jenniferholm, would be a good idea.